lib/index.ts

```diff
@@ -82,9 +104,12 @@ async function buildSubTreeRecursive(dep: string, depKeys: string[], lockFile: o
     depSubTree.depType = deps[dep].dev ? DepType.dev : DepType.prod;
     // repeat the process for dependencies of looked-up dep
     const newDeps = deps[dep].requires ? Object.keys(deps[dep].requires) : [];
+
     await Promise.all(newDeps.map(async (subDep) => {
-      depSubTree.dependencies[subDep] = await buildSubTreeRecursive(subDep, [...depKeys, dep], lockFile);
-    }));
+      depSubTree.dependencies[subDep] = await buildSubTreeRecursive(subDep,
```

Comment:
I think our allowed line length is too long, can we consider something smaller than 120?  Perhaps 100?

Reply:
I am fine with 100.